### DISCUSSION TITLE
Issuing of certificate?

### DISCUSSION THREAD

**Discussion by castledown** [chamilo/chamilo-lms#5975] [General] [answered]
I have version 1.11.28 and I seem to have done everything I should in order for a certificate to be generated, but when I trial it as a student, it says "You have not achieved any certificate just yet. Continue on your learning path to get one!" What am I missing...?

**Comment by ywarnier** [comment, marked as answer]
Hi @castledown 
Sorry for the late answer.
To generate a certificate, here are a few steps you need to check:
- in the assessments tool, clic on the pencil icon and make sure the "Generate certificate" checkbox is checked
- make sure the minimum score is achievable and is achieved by some learners (you can create a test user and go through the course yourself, then delete your user once you've tested it)
- make sure the certificate template works (doesn't seem to be related to your issue here) by going into the certificate section from the assessments homepage and clicking on the preview icon
- make sure the weight of whatever you are assessing is enough to actually get to the minimum score. If the minimum score of your certificate is 80% but the only item you have in the assessment is the learning path and it has a weight of "20" (on 100), the maximum score your students will ever get is 20% (100% of the learning path that is weighted at 20% of the total) and they will never reach the minimum score (of 80%)
- make sure you either use the learning path's "end page" (the F1-race finish flag) or that users go to the assessments tool (if they don't use any of those ways, the certificate is theoretically achieved, but they didn't trigger it so it is not generated)

I hope this helps.
